src/main/java/com/alkemy/ong/data/entity/ActivityEntity.java

```diff
@@ -0,0 +1,43 @@
+package com.alkemy.ong.data.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+
+@Getter
+@Setter
+@Entity
+@Table(name = "activities")
+@SQLDelete(sql = "UPDATE activities SET deleted = true WHERE id=?")
+@Where(clause = "deleted=false")
+public class ActivityEntity  {
+
+
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        private Long id;
+
+        private String name;
```

Comment:
A aquellos campos que en la base son not null le tienes que agregar esta identificación en la entity. @Column(nullable = false).
Vi que son varios en el script, por lo q t pido agregarles!

Reply:
Entiendo!. Ahí se los agrego